buildah.spec.rpkg

```diff
@@ -11,17 +11,19 @@
 
 %global with_debug 1
 
+# RHEL 8's default %%gobuild macro doesn't account for the BUILDTAGS variable, so we
+# set it separately here and do not depend on RHEL 8's go-srpm-macros package.
```

Comment:
If you take the prior
```suggestion
# set it separately here and do not depend on RHEL's go-srpm-macros package.
```

Reply:
@TomSweeneyRedHat nope, this is specifically for rhel8. rhel 9 is fine.